portfolio/src/main/webapp/script.js

```diff
@@ -177,9 +177,21 @@ function createCommentElement(comment) {
 
   const contentElement = document.createElement('span');
   contentElement.innerText = comment.content;
+  const dateElement = document.createElement('p');
+  dateElement.innerText = comment.currentDate;
 
   commentElement.appendChild(contentElement);
+  commentElement.appendChild(dateElement);
   return commentElement;
 }
 
+function getNumberOfComments() {
+  const numberOfCommentsMenu = document.getElementById("numberOfComments");
+  let numberOfComments = numberOfCommentsMenu.options[numberOfCommentsMenu.selectedIndex].value;
+  return numberOfComments;
+}
 
+function changeNumberOfComments() {
+  document.getElementById("commentsList").innerHTML = "";
```

Comment:
This line removing all the comments should probably move to inside loadComments() since there isn't really a case where you would want to load comments without removing the existing comments.